src/edu/washington/escience/myria/util/DeploymentUtils.java

```diff
@@ -158,30 +81,118 @@ public static void main(final String[] args) throws IOException {
   }
 
   /**
+   * 
+   * @param config a Myria configuration.
+   * @throws ConfigFileException if error occurred parsing config file.
+   */
+  public static void startMaster(final MyriaConfiguration config) throws ConfigFileException {
+    int restPort = Integer.parseInt(config.getRequired("deployment", MyriaSystemConfigKeys.REST_PORT));
+    String maxHeapSize = config.getOptional("runtime", MyriaSystemConfigKeys.MAX_HEAP_SIZE);
+    String sslStr =
+        MoreObjects.firstNonNull(config.getOptional("deployment", MyriaSystemConfigKeys.SSL), "false").toLowerCase();
+    boolean ssl = false;
+    if (sslStr.equals("true") || sslStr.equals("on") || sslStr.equals("1")) {
+      ssl = true;
+    }
+    String hostname = config.getHostnameWithUsername(MyriaConstants.MASTER_ID + "");
+    String workingDir = config.getWorkingDirectory(MyriaConstants.MASTER_ID + "");
+    startMaster(hostname, workingDir, maxHeapSize, restPort, ssl);
+  }
+
+  /**
+   * 
+   * @param config a Myria configuration.
+   * @param workerId worker ID.
+   * @throws ConfigFileException if error occurred parsing config file.
+   */
+  public static void startAWorker(final MyriaConfiguration config, final String workerId) throws ConfigFileException {
```

Comment:
Why can't this just be called `startWorker` since you've already overloaded `startMaster`?